examples/example3/example3_withmanufactured.ipynb

```diff
@@ -0,0 +1,679 @@
+    "- $\\Omega_{Cyto}: r \\in [0, 1) \\subset \\mathbb{R}^3$\n",
+    "- $\\Gamma_{PM}: r=1 \\subset \\mathbb{R}^3$\n",
+    "\n",
+    "We generate the parent mesh with appropriate markers using gmsh in the function `mesh_tools.create_spheres`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "308b6a58",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Base mesh\n",
+    "# 0 in second argument corresponds to no inner sphere\n",
+    "xSize, ySize, zSize = 2.0, 2.0, 2.0\n",
+    "domain = d.BoxMesh(d.Point(0.0, 0.0, 0.0), d.Point(xSize, ySize, zSize), 10, 10, 10)\n",
+    "cell_markers = d.MeshFunction(\"size_t\", domain, 3, 1)\n",
+    "facet_markers = d.MeshFunction(\"size_t\", domain, 2, 0)\n",
+    "for f in d.facets(domain):\n",
+    "    x, y, z = f.midpoint()[:]\n",
+    "    if np.isclose(x, 0.):\n",
+    "        facet_markers[f] = 10\n",
+    "    elif np.isclose(x, xSize):\n",
+    "        facet_markers[f] = 11\n",
+    "    elif np.isclose(y, 0.):\n",
+    "        facet_markers[f] = 12\n",
+    "    elif np.isclose(y, ySize):\n",
+    "        facet_markers[f] = 13\n",
+    "    elif np.isclose(z, 0.):\n",
+    "        facet_markers[f] = 14\n",
+    "    elif np.isclose(z, zSize):\n",
+    "        facet_markers[f] = 15\n",
+    "# domain, facet_markers, cell_markers = mesh_tools.create_spheres(curRadius, 0, hEdge=0.05, hInnerEdge=0.05)\n",
+    "# visualization.plot_dolfin_mesh(domain, cell_markers, facet_markers)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "3967a934",
+   "metadata": {},
+   "source": [
+    "In order to load this into a `ParentMesh` object, we need to first save it and then load it in using the `smart.mesh.ParentMesh` function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94a798bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Write mesh and meshfunctions to file\n",
+    "mesh_folder = pathlib.Path(f\"mesh_{curRadius:03f}/\")\n",
+    "mesh_folder.mkdir(exist_ok=True)\n",
+    "mesh_file = mesh_folder / \"DemoSphere.h5\"\n",
+    "mesh_tools.write_mesh(domain, facet_markers, cell_markers, filename=mesh_file)\n",
+    "# # Define parent mesh\n",
+    "parent_mesh = mesh.ParentMesh(\n",
+    "    str(mesh_file),\n",
+    "    mesh_filetype=\"hdf5\",\n",
+    "    name=\"parent_mesh\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "309e5e8a",
+   "metadata": {},
+   "source": [
+    "Define manufactured solution and associated reactions. To not make the choice of manufactured solution completely arbitrary, we have chosen it as a multiplication of three solutions to the PDE in 1D, then multiplied by some time-dependence governed by the time-scale of protein dephosphorylation in the cytosol."
```

Comment:
We should probably have these equations in Latex format here as well, to make it easy to show the reader how to go from mathematics to reactions in smart.